### ISSUE TITLE
how/where does it work?

### ISSUE THREAD

**Issue by CouldBeThis** [dreadnaut/bordercolors-d#9]
Hi, 

I installed this in TB 68.11 and I can see the add on. But how do I activate or configure it?

thanks for you time

**Comment by dreadnaut** [comment]
In Thunderbird 68, you can find the options in the Tools menu, under Tools \ Add-on Options.

![image](https://user-images.githubusercontent.com/584584/89890230-e8924c00-dbca-11ea-8cdb-90b743b116ae.png)

**Comment by CouldBeThis** [comment]
Thank you. 

Very weirdly, when I hover over the *Add-on Options* (actually *Preferences* in my version) item, no sub menus pops up. 

By way of troubleshooting I installed CompactHeader which is also visible in your screen cap and after restarting there is no change in behaviour. No sub menu. 

I tried disabling all add ons except BorderColurs D or CompactHeader. Also tried disabling absolutely all add ons. In no case was I able to produce a sub menu (even an empty one). 

Sooo I guess whatever this is isn't your problem; seems to be TB generally. Maybe I will investigate further when I have time. 

thanks for the help in any event
